modules/quality_control/php/quality_control_behavioral.class.inc

```diff
@@ -0,0 +1,169 @@
+<?php
+/**
+ * This file contains the NDB_Menu_Filter_media class
+ *
+ * PHP Version 7
+ *
+ * @category Module
+ * @package  Quality_Control
+ * @author   Liza Levitis <llevitis.mcin@gmail.com>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/CCNA/
+ */
+namespace LORIS\quality_control;
+
+/**
+ * Main class for quality control module corresponding to /quality_control/ URL
+ * Tools section of the LorisMenu.
+ *
+ * PHP Version 7
+ *
+ * @category Module
+ * @package  Quality_Control
+ * @author   Liza Levitis <llevitis.mcin@gmail.com>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/CCNA/
+ */
+class Quality_Control_Behavioral extends \NDB_Menu_Filter
+{
+    public $hasWritePermission = false;
+    public $AjaxModule         = true;
+    public $fieldOptions;
+
+    /**
+     * Check user permissions
+     *
+     * @param \User $user UserID
+     *
+     * @return bool
+     * @throws \ConfigurationException
+     */
+    function _hasAccess(\User $user) : bool
+    {
+        //create user object
+        return $user->hasPermission('data_team_helper');
+    }
+
+    /**
+     * Create a form to filter behavioral QC by various criteria
+     *
+     * @return void
+     * @throws \ConfigurationException
+     * @throws \DatabaseException
+     * @throws \Exception
+     */
+    function _setFilterForm(): void
```

Comment:
... This doesn't use a filter form so I don't think this is actually the right function. It think it's something more like `_setUpVariables`. Sorry!